applications/vanilla/models/class.discussionmodel.php

```diff
@@ -2138,8 +2138,8 @@ public function save($formPostValues, $settings = false) {
                     $stored = $this->getID($discussionID, DATASET_TYPE_OBJECT);
 
                     // Block Format change if we're forcing the formatter.
```

Comment:
You should update this description too since we're updating the code.

Reply:
done